src/components/CourseNavMenu/stylesheet.scss

```diff
@@ -0,0 +1,24 @@
+.course-nav-menu {
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    justify-content: flex-start;
+
+    .course-nav-button {
+        position: relative;
+        overflow: hidden;
+        justify-content: flex-start;
+
+        // Left-align the buttons with the close button
+        padding-left: 24px;
+
+        // Increase the size of the tap target
+        padding-top: 16px;
+        padding-bottom: 16px;
```

Comment:
Let's add a right padding too. The text in the tab isn't centered since it's only left-padded.